iot/api-client/mqtt_example/cloudiot_mqtt_example.py

```diff
@@ -87,6 +87,13 @@ def on_publish(unused_client, unused_userdata, unused_mid):
     print('on_publish')
 
 
+def on_message(unused_client, unused_userdata, message):
+    """Callback when the device receives a message on a subscription."""
+    payload = str(message.payload)
+    print "Received message '{}' on topic '{}' with Qos {}".format(
```

Comment:
print is a function, and please use single quotes for strings.

Reply:
Done

Reply:
@jonparrott Did I miss one?